src/fides/api/graph/traversal.py

```diff
@@ -387,7 +387,8 @@ def edge_ends_with_collection(_edge: Edge) -> bool:
                 ",".join([str(x) for x in remaining_node_keys]),
             )
             raise TraversalError(
-                f"Some nodes were not reachable: {','.join([str(x) for x in remaining_node_keys])}"
+                f"Some nodes were not reachable: {','.join([str(x) for x in remaining_node_keys])}",
+                [key.value for key in remaining_node_keys],
             )
```

Comment:
Why is this formatting and the one below changed?

Reply:
I don't think it's just the formatting, I added the failing node keys as the list of errors so I could get them when catching the exception

Reply:
ah! I missed the fidesops exception takes in a string as well as a list of errors!